### ISSUE TITLE
ImportError: No module named 'numpy.core.multiarray\r'

### ISSUE THREAD

**Issue by elinz** [tylerjereddy/pycon-2017#1]
Received "ImportError: No module named 'numpy.core.multiarray\r'" on various code block which run pickle on the following lines:

6.4.2) 3D
flu_coords = pickle.load(open('flu_coords.p','rb'), encoding='latin1') # was pickled in py2, so encoding issues

8.3) Practical Problem Solving with scipy.spatial.Voronoi()
array_pump_coords = pickle.load(open('array_pump_data.p','rb'), encoding='latin1')

**Comment by tylerjereddy** [comment]
Hmm pickle stuff can be tricky. Are you running on Windows? I should probably use the [`npy` format](https://docs.scipy.org/doc/numpy-1.12.0/reference/generated/numpy.load.html) instead of pickle for these.

**Comment by elinz** [comment]
Yes, I am running Python on windows 64 bit OS (Windows 8.1 Enterprise). Thank you.

**Comment by tylerjereddy** [comment]
Can you try executing the latest version of the notebook if you have a minute? I've replaced all usage of pickle with `np.load` and the equivalent `npy` files, which should increase compatibility.
